client/app/domain/options/delete/domain-options-delete.controller.js

```diff
@@ -0,0 +1,46 @@
+angular.module('controllers').controller(
+  'controllers.Domain.Options.Delete',
+  class DomainDnsLockCtrl {
```

Comment:
You could extract the class

Reply:
This class is not a reusable and is a controller class. I guess this class can be left as such?